netty/src/main/java/io/grpc/transport/netty/NettyClientHandler.java

```diff
@@ -192,7 +213,7 @@ private void onRstStreamRead(int streamId)
   public void channelInactive(ChannelHandlerContext ctx) throws Exception {
     try {
       // Fail any streams that are awaiting creation.
-      Status goAwayStatus = goAwayStatus().withDescription("network channel closed");
+      goAwayStatus(goAwayStatus().withDescription("Network channel closed"));
```

Comment:
this is overwriting an earlier, and possibly more useful description. Could augment it instead